modules/system/rolls/cast-test.js

```diff
@@ -105,15 +123,27 @@ export default class CastTest extends TestWFRP {
         }
         //@/HOUSE
       }
+      //@/HOUSE
+      if (this.preData.overchannelling > 0) { 
+        this.result.tooltips.miscast.push(game.i18n.localize("CHAT.OverchannellingMiscast"))
+        miscastCounter++;
+      }
+      //@/HOUSE
     }
     else if (slOver < 0) // Successful test, but unable to cast due to not enough SL
     {
       this.result.castOutcome = "failure"
       this.result.description = game.i18n.localize("ROLL.CastingFailed")
-
+      //@/HOUSE
+      if (this.preData.overchannelling > 0) { 
+        this.result.tooltips.miscast.push(game.i18n.localize("CHAT.OverchannellingMiscast"))
+        miscastCounter++;
+      }
+      //@/HOUSE
```

Comment:
Can you add  `game.wfrp4e.utility.logHomebrew("overchannelling")`

Reply:
done